### PR TITLE
Dev console

### PR DESCRIPTION
**Pull request by minshallj**

### CODE REVIEW

**Comment by minshallj** [comment]
based off non-squashed master, will delete all commits from squash

**Comment by FuriousJulius** [comment]
test this please

**Comment by FuriousJulius** [comment]
test this please

**Comment by FuriousJulius** [comment]
test it again jenkins

**Comment by FuriousJulius** [comment]
test this please

File: lg_sv/test/sv_server.test [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <launch>
-    <test test-name="test_sv_server" pkg="lg_sv" type="test_sv_server.py" />
+    <test test-name="test_sv_server" pkg="lg_sv" type="test_sv_server.py" time-limit="200.0" />
```
Comment (FuriousJulius) [inline lg_sv/test/sv_server.test:2]: test this ?

**Comment by FuriousJulius** [comment]
test this please

**Comment by FuriousJulius** [comment]
rebuild

**Comment by FuriousJulius** [comment]
rebuild

**Comment by minshallj** [comment]
rebuild

**Pull request closed without merging**